File: python/14.py
##14
userChoice = int(input())
## check if it matches
if userChoice < 6:
    result = "Your number is less than 6, or more than 10"
elif userChoice > 10:
    result = "Your number is less than 6, or more than 10"
else:
    result = "Your number is NOT less than 6, or more than 10"
print(result)

File: js/14.js
//14
// import things
const prompt = require("prompt-sync")({ sigint: true });
// get the input
let userChoice = prompt();
// selection statement here we wo we wo
if (userChoice < 6 || userChoice > 10){
    console.info("Your number is less than 6, or greater than 10")
} else {
    console.warn("Your number NOT is less than 6, or greater than 10")
}